frontend/src/components/hub/HubTabsNavigation.tsx

```diff
@@ -10,7 +10,7 @@ import HubsDropDown from "../indexPage/hubsSubHeader/HubsDropDown";
 
 const useStyles = makeStyles((theme) => ({
   root: {
-    background: theme.palette.primary.main,
+    background: theme.palette.background.default,
```

Comment:
The section of the post that you posted doesn't reference primary/background at all or am I missing something?
It seems to be an argument to use `background.main` but I don't see how it would be an argument to use `primary.main`